test/image.bats

```diff
@@ -293,3 +294,29 @@ function teardown() {
 
 	cleanup_images
 }
+
+@test "run container in pod with crun-wasm enabled" {
+	if [ -z "$CRUN_WASM_BINARY" ] || [[ "$RUNTIME_TYPE" == "vm" ]]; then
+		skip "crun-wasm not installed or runtime type is VM"
+	fi
+	cat << EOF > "$CRIO_CONFIG_DIR/99-crun-wasm.conf"
+[crio.runtime]
+default_runtime = "crun-wasm"
+
+[crio.runtime.runtimes.crun-wasm]
+runtime_path = "/usr/bin/crun"
+
+[crio.runtime.runtimes.crun-wasm.platform_runtime_paths]
+"wasi/wasm32" = "/usr/bin/crun-wasm"
```

Comment:
doesn't match what you generate in the template above

Reply:
Fixed